plugin/output/s3/s3.go

```diff
@@ -495,14 +416,40 @@ func (p *Plugin) uploadWork() {
 		for {
 			err := p.uploadToS3(compressed)
 			if err == nil {
-				p.logger.Infof("successfully uploaded object: %s", compressed)
+				p.logger.Infof("successfully uploaded object: %v", compressed)
+
+				// send message to commiter
+				if p.config.CommitCfg != nil {
+					commiterData := make(map[string]interface{})
+					if p.config.CommitCfg.FirstTimestampFieldForTrack != "" {
+						commiterData[p.config.CommitCfg.FirstTimestampFieldForTrack] = compressed.firstTimestamp
+					}
+					if p.config.CommitCfg.LastTimestampFieldForTrack != "" {
+						commiterData[p.config.CommitCfg.LastTimestampFieldForTrack] = compressed.lastTimestamp
+					}
+					if p.config.CommitCfg.BucketNameFieldForTrack != "" {
+						commiterData[p.config.CommitCfg.BucketNameFieldForTrack] = compressed.bucketName
+					}
+					if p.config.CommitCfg.S3UrlNameFieldForTrack != "" {
+						commiterData[p.config.CommitCfg.S3UrlNameFieldForTrack] = compressed.fileName
+					}
+					err := p.commiterWrapper.CommitUpload(commiterData, p.config.CommitCfg.ConstantCommitMessagePart)
+					if err != nil {
+						p.logger.Fatalf("can't commit message of uploading: %s", err.Error())
+					}
```

Comment:
Is this a desired logic - if we can not write S3 file meta once we just pass?
If so I think that `commit` is a wrong word. It is just writing meta smth like trySaveMeta()

Reply:
meta seems good